worker/exec/konnector_test.go

```diff
@@ -330,16 +330,21 @@ echo "{\"type\": \"toto\", \"message\": \"COZY_URL=${COZY_URL}\"}"
 		}()
 
 		wg.Wait()
-		wg.Add(1)
-		acc := &account.Account{DefaultFolderPath: "/Administrative/toto"}
-		assert.NoError(t, couchdb.CreateDoc(inst, acc))
+
+		acc := &account.Account{}
 		defer func() { _ = couchdb.DeleteDoc(inst, acc) }()
+
+		// Folder is created from DefaultFolderPath
+		wg.Add(1)
+		acc.DefaultFolderPath = "/Administrative/toto"
+		require.NoError(t, couchdb.CreateDoc(inst, acc))
```

Comment:
I don't understand why the defer for deleting the doc is before the creation of it.

Reply:
Since I'm using the same `acc` variable I decided to call the add the `defer` and then manipulate the account. Is that a bad habit?

Reply:
Well, as we ignore the error on `DeleteDoc`, it works, but I don't think it is a good practice. I try to put the defer for cleaning just after the line that create the thing. Here, the deferred `DeleteDoc` would come just after `CreateDoc`.